test/lit/passes/extract-function.wast

```diff
@@ -1,71 +1,84 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
 ;; RUN: foreach %s %t wasm-opt --extract-function=foo -S -o - | filecheck %s
 ;; RUN: foreach %s %t wasm-opt --extract-function --pass-arg=extract-function@foo -S -o - | filecheck %s
 ;; RUN: foreach %s %t wasm-opt --extract-function-index=0 -S -o - | filecheck %s
 ;; RUN: foreach %s %t wasm-opt --extract-function-index --pass-arg=extract-function-index@0 -S -o - | filecheck %s
 
-;; CHECK:      (module
```

Comment:
Were we already supporting multiple modules within a file in lit tests? I thought that splitting and splicing was supported in our old tests but not here, but I might have missed a few PRs

Reply:
Yes, but this is the only test that uses it so far. This test was introduced along with the `foreach` tool in https://github.com/WebAssembly/binaryen/pull/3939.